### PR TITLE
Detailing `-chdir` behavior with `terraform.d/plugin` folders

### PR DESCRIPTION
**Pull request by fnickels**
Fixes Bug reported in https://github.com/hashicorp/terraform/issues/31442

This PR is a 4th alternate implementation - documentation change to match code behavior.

Use instead if the following PRs are not workable:
* https://github.com/hashicorp/terraform/pull/31456 
* https://github.com/hashicorp/terraform/pull/31452 
* https://github.com/hashicorp/terraform/pull/31443

### CODE REVIEW

**Comment by crw** [comment]
Thanks for these documentation changes. I'll run these past the docs team for review.

File: website/docs/cli/config/config-file.mdx [hunk 1]
```diff
@@ -311,7 +311,9 @@ the operating system where you are running Terraform:
 
 If a `terraform.d/plugins` directory exists in the current working directory
 then Terraform will also include that directory, regardless of your operating
-system.
+system. *The behavior is slightly different when using `-chdir` with the `init` command,
+the launch directory, not the directory specified with `-chdir`, is used
+for checking for the existence of the `terraform.d/plugins` directory.*
```
Comment (laurapacilio) [inline website/docs/cli/config/config-file.mdx:316]:
```suggestion
system. This behavior changes when you use the `-chdir` option with the `init` command. In that case, Terraform checks for the `terraform.d/plugins` directory in the launch directory and not in the directory you specified with `-chdir`.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Hi @fnickels - thanks for adding this! I'm making a suggested edit to help with clarity and flow. Please let me know if I have misunderstood your intended meaning. Do you feel comfortable merging it in?

@crw After this, I would like a quick engineering technical 👍 as well - this is a case I'm not super familiar with and I want to make sure engineering is okay with this. Especially since there has been discussion about `-chdir` in other threads. Thank you!

**Review by laurapacilio — APPROVED** [review]
🚀  @crw - over to eng. for a final 👍

**Comment by crw** [comment]
For future reference: I believe this comment confirms the behavior as described in these docs changes. However, I'll queue this up for final review. 

https://github.com/hashicorp/terraform/issues/31442#issuecomment-1185646259

**Pull request merged**